app/src/components/account/UserMenu.vue

```diff
@@ -49,22 +53,42 @@
           <div class="address-item">
             <img
               class="network-icon"
-              :src="address.icon"
+              :src="address.icon || `/img/networks/${address.networkId}.png`"
```

Comment:
some of my sessions / accounts don't have an icon or networkId

Reply:
that is why I am enriching them in the computed `addresses`, to make sure they all have a networkId field